barpyrus/trayer.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
+
+from barpyrus.widgets import Widget
+from barpyrus.core import EventInput
+from Xlib.display import Display, X
+
+class TrayerWatch(EventInput):
+    def __init__(self, args = { }):
+        command = [ 'trayer' ]
+        self.default_args = {
+            'edge': 'top',
+            'align': 'right',
+            'widthtype': 'request',
+            'expand': 'true',
+            'SetDockType': 'true',
+            'transparent': 'true',
+            'alpha': '0',
+            'height': '16',
+            'margin': '0',
+            'tint': '0x29b2e',
+        }
+        for key,val in args.items():
+            self.default_args[key] = val
```

Comment:
You can just do `self.default_args.update(args)` instead.

Reply:
[x]